app-ios/Modules/Sources/Timetable/TimetableDayHeader.swift

```diff
@@ -9,39 +9,62 @@ struct TimetableDayHeader: View {
     let onSelect: (DroidKaigi2023Day) -> Void
 
     var body: some View {
-        HStack(spacing: 8) {
-            ForEach(
-                [DroidKaigi2023Day].fromKotlinArray(DroidKaigi2023Day.values()),
-                id: \.ordinal
-            ) { (day: DroidKaigi2023Day) in
-                Button {
-                    onSelect(day)
-                } label: {
-                    VStack(spacing: 0) {
-                        Text(day.name)
-                            .font(Font.system(size: 12, weight: .semibold))
-                        Text("\(day.dayOfMonth)")
-                            .font(Font.system(size: 24, weight: .semibold))
-                            .frame(height: 32)
+        ZStack {
+            AssetColors.Surface.surface.swiftUIColor
+            HStack(spacing: 8) {
+                ForEach(
+                    [DroidKaigi2023Day].fromKotlinArray(DroidKaigi2023Day.values()),
+                    id: \.ordinal
+                ) { (day: DroidKaigi2023Day) in
+                    Button {
+                        onSelect(day)
+                    } label: {
+                        VStack(spacing: 0) {
+                            Text(day.name)
+                                .font(Font.system(size: 12, weight: .semibold))
+                            Text("\(day.dayOfMonth)")
+                                .font(Font.system(size: 24, weight: .semibold))
+                                .frame(height: 32)
+                        }
+                        .padding(4)
+                        .frame(maxWidth: .infinity)
+                        .foregroundStyle(
+                            selectedDay == day
+                            ? AssetColors.Primary.onPrimary.swiftUIColor
+                            : AssetColors.Surface.onSurfaceVariant.swiftUIColor)
                     }
-                    .padding(4)
-                    .frame(maxWidth: .infinity)
-                    .foregroundStyle(
-                        selectedDay == day
-                        ? AssetColors.Primary.onPrimary.swiftUIColor
-                        : AssetColors.Surface.onSurfaceVariant.swiftUIColor)
-                    .background(
-                        selectedDay == day
-                        ? AssetColors.Primary.primary.swiftUIColor
-                        : Color.clear
-                    )
-                    .clipShape(Capsule())
+                }
+            }
+            .padding(.horizontal, 16)
+            .padding(.vertical, 12)
+            .background(alignment: .center) {
+                GeometryReader { geometry in
+                    Capsule()
+                        .fill(AssetColors.Primary.primary.swiftUIColor)
+                        .frame(width: calcButtonWidth(width: geometry.size.width), height: 56)
```

Comment:
Thank you for your first PR to this repo! Looks awesome to me 🤩

I have 1 request.
If possible, can you define some constants used in this component to make it easier to understand what are these magic numbers? 🙏

Reply:
@tkhs0604 

Thank you for request:bow:
The following values ​​are related to calculations, so they are defined separately.

```swift
// Define margin values to calculate holizontal position for capsule rectangle
private let buttonAreaLeadingMargin = 16.0
private let buttonTrailingMargin = 16.0
private let betweenButtonMargin = 8.0

// Define all button count to calculate holizontal position for capsule rectangle
private var buttonsCount: Int {
    [DroidKaigi2023Day]
        .fromKotlinArray(DroidKaigi2023Day.values())
        .count
}
```

By the way, refactor the methods related to computation.

Reply:
Thank you, they are so readable! Let me check again 🙆‍♂️